packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -15,7 +15,7 @@ import { getRole } from './baseSelectors';
 import { CallParticipantListParticipant } from '@internal/react-components';
 import { _isRingingPSTNParticipant, _updateUserDisplayNames } from './utils/callUtils';
 import { memoizedConvertAllremoteParticipants } from './utils/participantListSelectorUtils';
-/* @conditional-compile-remove(rooms) */
+/* @conditional-compile-remove(raise-hand) */
```

Comment:
If its used for both features, and both to the conditional compilation line
```suggestion
/* @conditional-compile-remove(rooms) */ /* @conditional-compile-remove(raise-hand) */
```

Reply:
(for this line and throughout)